avatars/animationHelpers.js

```diff
@@ -1203,6 +1233,17 @@ export function getFirstPersonCurves(vrmExtension) {
   }
 }
 
+export function trimClip(clip, startTime, endTime) {
```

Comment:
This should be in the animations baker.

Reply:
Moved `trimClip()` from runtime to baker. https://github.com/webaverse/app/pull/2791/commits/f056c61d8d6daa123fee7d9c482dc0ed83211ada